python/common/org/Python.java

```diff
@@ -602,12 +602,44 @@ public static void initializeModule(java.lang.Class cls, java.util.Map<java.lang
                     "iteration.  The enumerate object yields pairs containing a count (from\n" +
                     "start, which defaults to zero) and a value yielded by the iterable argument.\n" +
                     "enumerate is useful for obtaining an indexed list:\n" +
-                    "       (0, seq[0]), (1, seq[1]), (2, seq[2]), ...\n"
+                    "(0, seq[0]), (1, seq[1]), (2, seq[2]), ...\n",
+            args = {"items"},
+            default_args = {"start"}
     )
-    public static org.python.Iterable enumerate() {
-        throw new org.python.exceptions.NotImplementedError("Builtin function 'enumerate' not implemented");
+    public static org.python.Iterable enumerate(org.python.Object items, org.python.Object start) {
+        try {
+            org.python.Object index = new org.python.types.Int(0);
+            if (start == null) {
+                start = new org.python.types.Int(0);
```

Comment:
This statement doesn't seem very necessary because `start` isn't used after this – `index` is, and that's already initialised to 0 a few lines above this, or in the below `else` block.

Perhaps this line should be removed, and the if conditional changed to `start != null`?